README.md

```diff
@@ -24,6 +24,8 @@
 
 <p align="center">
   <img alt="Omni screnshoot for Firefox" src="./screenshot.png">
+  <br/><br/>
+  <img alt="Omni screnshoot for Thunderbird" src="./screenshot2.png">
```

Comment:
I think this image is not necessary. Let's remove it and keep the info about Thunderbird just on the title and description. Don't forget to remove the `screenshot2` file.

Reply:
Fixed in 96566293b657b52b1ff0e1eb7dd1e94ebf697082